src/ast/dep.rs

```diff
@@ -369,6 +369,32 @@ impl Visit for DependencyCollector<'_> {
       );
     }
   }
+
+  fn visit_ts_module_decl(&mut self, node: &ast::TsModuleDecl) {
+    if let Some(id_str) = node.id.as_str() {
+      let value_str = id_str.value.as_str();
+      if !value_str.contains('*')
+        || value_str.starts_with("./")
+        || value_str.starts_with("../")
+        || value_str.starts_with('/')
+      {
+        let leading_comments = self.get_leading_comments(node.start());
+        self.items.push(
+          StaticDependencyDescriptor {
+            kind: StaticDependencyKind::MaybeTsModuleAugmentation,
+            leading_comments,
+            range: id_str.range(),
+            specifier: id_str.value.clone(),
+            specifier_range: id_str.range(),
+            import_attributes: Default::default(),
+            is_side_effect: false,
+          }
+          .into(),
+        );
+      }
+    }
+    node.visit_children_with(self);
```

Comment:
Do the children need to be visited? Maybe we can skip it? I forget what's allowed in a module augmentation block.

Reply:
There could be JSDoc `import()`s, at the very least